### DISCUSSION TITLE
Change Limit reservable bandwidth value?

### DISCUSSION THREAD

**Discussion by fabcard** [farag2/Sophia-Script-for-Windows#226] [Q&A] [answered]
Hello everyone,
First, I would like to thank @farag2 for the amazing work. Those scripts help a lot!
I have a request if possible, please. Can you add an option to set the "Limit reservable bandwidth" to 0 or any other value as shown here https://www.youtube.com/watch?v=7u1miYJmJ_4 min 2.37?
Thanks in advance.

**Comment by farag2** [comment, marked as answer]
I'll think it out. I've read about this key.

**Reply by fabcard** [reply]
Thank you!

**Comment by farag2** [comment]
That's how I thought from the beginning (when I saw this video half a year ago). Of course, this is a bullshit. He's only making views.
https://www.howtogeek.com/730147/windows-10-myth-dont-touch-limit-reservable-bandwidth/

So don't think that Windows limits your bandwith. :)

**Reply by fabcard** [reply]
Oh, I see. Thank you, @farag2, for your reply.
Feel free to delete this post if you want.
I'll be looking for more of your amazing scripts.
Again, thank you.

**Reply by farag2** [reply]
Cross your fingers and wait for SophiApp. :P
